### PR TITLE
Add the release note for 1.5.0

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

**Review by ikhoon — APPROVED** [review]
Thanks! @minwoox

File: site/src/pages/release-notes/1.5.0.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+date: 2021-02-13
+---
+
+## 🌟 New features
+
+- You can now handle asynchronous Servlets in <type://JettyService>. #3320 #33311
```
Comment (trustin) [inline site/src/pages/release-notes/1.5.0.mdx:7]: handle -> run ?

Comment (trustin) [reply site/src/pages/release-notes/1.5.0.mdx]: #33311 -> #3331

**Review by trustin — APPROVED** [review]
Thank you, @minwoox !

**Pull request merged**